testkit/tests/counter/counter.rs

```diff
@@ -222,7 +222,7 @@ impl CounterApi {
     artifact_name = "counter-service",
     artifact_version = "1.0.0",
     proto_sources = "crate::proto",
-    service_interface = "CounterServiceInterface"
+    implements("CounterServiceInterface")
```

Comment:
Just thinking out loud: maybe it makes sense to deduce interface name automatically if no `implements` parameter was supplied?..
Because now we've got a bunch of boilerplate segments like `implements("StructNameInterface"))] pub struct StructName`.

On the one hand, it will maybe make things somewhat more implicit, but on the other hand, if user doesn't use multiple interfaces in his code, why bother him with that?

Reply:
In my opinion deduction is antipattern.

Reply:
Why so?